src/python_tools/plotting/ssl_wrapper_plot_helpers.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+from bokeh.plotting import figure
+from bokeh.models import ColumnDataSource, LabelSet
+from software.proto.messages_robocup_ssl_geometry_pb2 import (
+    SSL_FieldCircularArc,
+    SSL_FieldLineSegment,
+)
+from software.proto.messages_robocup_ssl_detection_pb2 import (
+    SSL_DetectionBall,
+    SSL_DetectionRobot,
+)
+from software.proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket
+from typing import Iterable
+
+ROBOT_MAX_RADIUS = 0.09
```

Comment:
I was wondering whether it matters that these constants match the constants defined in the other parts of the code base (ROBOT_MAX_RADIUS and BALL-RADIUS)

Reply:
this is good point, can we use python-cpp bindings to keep them consistent?

Reply:
the bindings are too much of a pain to set up just for this

Reply:
the ball radius is currently an arbitrary value but the ROBOT_MAX_RADIUS is copypasted from the cpp code. It depends on what users use this for, for most use cases I think a "good enough" value works.